adyenv6core/src/com/adyen/v6/service/DefaultAdyenPaymentService.java

```diff
@@ -328,6 +332,23 @@ public PaymentsResponse getPaymentDetailsFromPayload(HashMap<String, String> det
         return paymentsResponse;
     }
 
+    @Override
+    public String getOriginKey(String originDomain) throws ApiException, IOException {
+        CheckoutUtility checkoutUtility = new CheckoutUtility(client);
+        OriginKeysRequest originKeysRequest = new OriginKeysRequest();
+        String originkey = "";
+        ArrayList<String> originDomains = new ArrayList<>(Arrays.asList(originDomain));
+
+        originKeysRequest.setOriginDomains(originDomains);
+        LOG.debug(originKeysRequest);
```

Comment:
is this not already logged in the library ?

Reply:
no, we don't do any logging in library.